### PR TITLE
Changes to get_parser 

### PR DESCRIPTION
**Pull request by RayStick**
When doing user testing, I first interpreted 'time points' to mean samples in my data, and got errors with using phys2bids because of that. I realized that it meant TRs fairly quickly, but I've suggested a small change to the wording in the help text to help that be clearer to people

### CODE REVIEW

**Comment by smoia** [comment]
@RayStick , can I use this PR of yours as an excuse to work a bit on the get_parser?
could you expand the description of the options (even all of them) to the point you think it's very clear what the program is doing - and possibly add all the default values at the end?

**Comment by RayStick** [comment]
Yes, I can do that.

**Comment by RayStick** [comment]
I am first trying to get some testing done. Once I've managed to get some things to work, I will find it easier to help write these things

**Comment by RayStick** [comment]
Ready to review

File: phys2bids/cli/run.py [hunk 1]
```diff
@@ -22,32 +22,39 @@ def _get_parser():
     required.add_argument('-in', '--input-file',
                           dest='filename',
                           type=str,
-                          help=('The name of the acq file, with or without '
-                                'extension.   Must be an .acq file!'),
+                          help='The name of the acquisition file, with or '
```
Comment (smoia) [inline phys2bids/cli/run.py]:
```suggestion
                          help='The name of the file containing physiological data, with or '
```

File: phys2bids/cli/run.py [hunk 2]
```diff
@@ -57,49 +64,52 @@ def _get_parser():
...
-                          help='The number corresponding to the channels to process.',
+                          help='The column numbers of  the channels to process. '
+                               'Default is to process all channels.',
                           default=None)
     optional.add_argument('-ntp', '--numtps',
                           dest='num_timepoints_expected',
                           type=int,
-                          help='Number of expected timepoints.',
+                          help='Number of expected timepoints (TRs). '
+                               'Default is 0.',
```
Comment (smoia) [inline phys2bids/cli/run.py]:
```suggestion
                               'Default is 0., Which means the control of first trigger won't happen.',
```

File: phys2bids/phys2bids.py [hunk 3]
```diff
@@ -17,8 +17,9 @@
 
 The project is under development.
 
-At the very moment, it assumes all the extracted channels from a file
-have the same sampling freq.
+At the very moment, it assumes:
+-  all the extracted channels from the input file have the same sampling freq.
```
Comment (smoia) [inline phys2bids/phys2bids.py]: Actually this is not true anymore!
```suggestion

```

File: phys2bids/cli/run.py (hunk 2 above)
Comment (RayStick) [reply phys2bids/cli/run.py]: "Control of the first trigger" is not clear to me

File: phys2bids/phys2bids.py (hunk 3 above)
Comment (RayStick) [reply phys2bids/phys2bids.py]: Oh, great!

File: phys2bids/cli/run.py (hunk 2 above)
Comment (smoia) [reply phys2bids/cli/run.py]: The guess of the start timing of the neuroimaging acquisition?

Comment (RayStick) [reply phys2bids/cli/run.py]: Ah I see what you mean now.  What about: "Default is 0, which means the estimate of when the neuroimaging acquisition started will be wrong."

Comment (smoia) [reply phys2bids/cli/run.py]: It's not going to be wrong, but it's not going to be performed, hence your file starting time is 0.
`the estimate of when the neuroimaging acquisition started will not take place.`?

Comment (RayStick) [reply phys2bids/cli/run.py]: Ah I seeeeee now! 
"Default 0. Note: the estimation of when the neuroimaging acquisition started cannot take place with this default"

**Comment by smoia** [comment]
@RayStick , status on this?

**Comment by RayStick** [comment]
I'm happy with it if you are! 
Was I meant to press a button? :)

File: phys2bids/cli/run.py (hunk 2 above)
Comment (smoia) [reply phys2bids/cli/run.py]: Can you change this in the file? After that, we can merge!

Comment (RayStick) [reply phys2bids/cli/run.py]: Oh I see. Yep

**Pull request merged**